scripts/zones/Grand_Palace_of_HuXzoi/mobs/DE_Quasilumin.lua

```diff
@@ -0,0 +1,307 @@
+-----------------------------------
```

Comment:
Can we move the contents of this file into a helper in this zones folder, or into the triggering zone or npc lua?

Reply:
Failing that, some comments explaining this is a dynamic entity and this is the naming convention would be welcome. Its a little arcane (my bad)

Reply:
Sure I don't mind doing either of these. Can you explain the motivation so I can understand better? Thanks!

Reply:
@zach2good think you could go into some details? :)

Reply:
I don't think the average person poking around in scripting is going to know that these Quasis are DE's, or necessarily what a DE is or does. The name is defined here:

https://github.com/LandSandBoat/server/blob/base/src/map/lua/lua_zone.cpp#L272

I think attaching the logic at the point of creation, or adding some very clear comments about why this file exists and is named the way it is would help out anyone whos poking around in here

Reply:
Thanks! 👍